### PR TITLE
udpate doc of groupby functions

### PR DESCRIPTION
**Pull request by davidstrouk**
some doc i wrote a while ago and forgot to push

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -5,3 +5,4 @@ max-line-length = 88
 extend-ignore =
     # See https://github.com/PyCQA/pycodestyle/issues/373
     E203,
+ignore = W605
```
Comment (uriva) [inline .flake8:8]: why is this needed?

Comment (davidstrouk) [reply .flake8:8]: "Return a mapping `{y: {x s.t. y \in f(x)}}, where `x \in it`. `"

when pre-commiting:
`gamla/graph.py:96:40: W605 invalid escape sequence '\i'`
